src/new-client/src/components/DraggableTearOut/DraggableTearOut.tsx

```diff
@@ -0,0 +1,120 @@
+import styled from "styled-components"
+import { RefObject, useState } from "react"
+import { tearOutSection, Section } from "@/App/TearOutSection/state"
+import { canDrag } from "@/components/DraggableTearOut/canDrag"
+import { tearOut } from "@/App/LiveRates/Tile/TearOut/state"
+
+export const DragWrapper = styled.div`
+  height: 100%;
+`
+
+type DraggableSectionTearOutProps = React.PropsWithChildren<{
+  section: Section
+  disabled?: boolean
+}>
+
+type DraggableTileTearOutProps = React.PropsWithChildren<{
+  symbol: string
+  tileRef: RefObject<HTMLDivElement>
+  disabled?: boolean
+}>
+
+type DraggableTearOutGenericProps = React.PropsWithChildren<{
+  dragHandler: () => void
+}>
+
+export const DraggableSectionTearOut: React.FC<DraggableSectionTearOutProps> = (
+  props: DraggableSectionTearOutProps,
```

Comment:
Just a minor style point - 
If you define a component's props with `React.FC<Props>` then you do not need to type the props argument `(props: Props)`.

Reply:
Fixed, now they only receive (props) without the mandatory type